docker/openemr/7.0.1/openemr.sh

```diff
@@ -391,6 +391,13 @@ fi
 if [ "$XDEBUG_IDE_KEY" != "" ] ||
    [ "$XDEBUG_ON" == 1 ]; then
    sh xdebug.sh
+else
+   # Configure opcache jit if Xdebug is not being used
+   if [ ! -f /etc/php-opcache-jit-configured ]; then
+      echo "opcache.jit_buffer_size=100M" >> /etc/php81/php.ini
+      echo "opcache.jit=tracing" >> /etc/php81/php.ini
+      touch /etc/php-opcache-jit-configured
+   fi
 fi
```

Comment:
Recommend doing the following in above if/else clause (and then no changes are needed in the php.ini file):
```
if [ "$XDEBUG_IDE_KEY" != "" ] ||
   [ "$XDEBUG_ON" == 1 ]; then
   sh xdebug.sh
   #also need to turn off opcache since it can not be turned on with xdebug
   if [ ! -f /etc/php-opcache-jit-configured ]; then
      echo "opcache.enable=0" >> /etc/php81/php.ini
      touch /etc/php-opcache-jit-configured
   fi
else
   # Configure opcache jit if Xdebug is not being used (note opcache is already on, so just need to add setting(s) to php.ini that are different from the default setting(s))
   if [ ! -f /etc/php-opcache-jit-configured ]; then
      echo "opcache.jit_buffer_size=100M" >> /etc/php81/php.ini
      touch /etc/php-opcache-jit-configured
   fi
fi
```

Reply:
Just confirming, it's okay to set php.ini variables multiple times and the last one governs? No need to replace the existing setting in the file?

Reply:
yes, that is my understanding. note we don't even need to touch php.ini since the setting changes are minimal (note could do the `opcache.revalidate_freq=60` here if wish also).